### ISSUE TITLE
[Feature Request] Time Zone Functionality

### ISSUE THREAD

**Issue by heviiguy** [nextcloud/calendar#164]
As one who routinely (when contract-hunting has been fruitful) jumps across many Time Zones, I really appreciate the "international" time functionality. However, I go to some pretty weird places. Not many of them are included in the calendar Time Zone drop-down list. Hence, I'm often left at a loss as to which city I must select to represent the applicable time zone!
Selecting a Time Zone from a basic map would be the ideal solution to this problem. Penultimately, grouping the cities by Time Zone would be quite helpul.

**Comment by georgehrke** [comment]
This project looks quite neat, but is apparently pretty dead :/

https://github.com/quicksketch/timezonepicker

**Comment by heviiguy** [comment]
Indeed. Just the kind of functionality which I had envisioned! Does being “dead” mean that this option is not available for us pursue?

**Comment by georgehrke** [comment]
With dead I mean, that the last commit was almost two years ago and there are issues / pull requests no one ever responded to.

The project is rather small and has a decent code quality.
But the map data is extremely outdated. E.g. it fails when I try to select south sudan.
So I'll look into how to update the map data and consider forking.
